test/system/250-systemd.bats

```diff
@@ -53,7 +52,6 @@ function service_setup() {
 
 # Helper to stop a systemd service running a container
 function service_cleanup() {
-    local status=$1
```

Comment:
I have no idea what this was intended to accomplish, but it doesn't, and hence is misleading.